src/v/redpanda/admin_server.cc

```diff
@@ -2893,6 +2935,26 @@ void admin_server::register_cluster_routes() {
 
           co_return ss::json::json_return_type(ret);
       });
+
+    register_route<superuser>(
+      ss::httpd::cluster_json::cancel_all_partitions_reconfigurations,
+      [this](std::unique_ptr<ss::httpd::request> req)
+        -> ss::future<ss::json::json_return_type> {
+          vlog(
+            logger.info,
+            "Requested cancellation of all ongoing partition movements");
+
+          auto res = co_await _controller->get_topics_frontend()
+                       .local()
+                       .cancel_moving_all_partition_replicas(
```

Comment:
Should we also have force-abort versions of these endpoints? (probably can be addressed in a followup PR)

Reply:
ok, i will add them in a follow up PR.

Reply:
added #5782